delphi/translators/for2py/genCode.py

```diff
@@ -5,249 +5,370 @@
 
 class PrintState:
     def __init__(self, sep=None, add=None):
-        self.sep = sep if sep != None else "\n"
-        self.add = add if add != None else "    "
+        self.sep = sep if sep is not None else "\n"
+        self.add = add if add is not None else "    "
 
     def copy(self, sep=None, add=None):
         return PrintState(
-            self.sep if sep == None else sep, self.add if add == None else add
+            self.sep if sep is None else sep, self.add if add is None else add
         )
 
+class genCode:
+    def __init__(self,
+                 lambda_string=""
+                 ):
+        self.lambda_string = lambda_string
+
+        self.process_lambda_node = {
+            "ast.FunctionDef": self.process_function_definition,
+            "ast.arguments": self.process_arguments,
+            "ast.arg": self._process_arg,
+            "ast.Load": self._process_load,
+            "ast.Store": self._process_store,
+            "ast.Index": self.process_index,
+            "ast.Num": self._process_num,
+            "ast.List": self.process_list_ast,
+            "ast.Str": self._process_str,
+            "ast.For": self.process_for,
+            "ast.If": self.process_if,
+            "ast.UnaryOp": self.process_unary_operation,
+            "ast.BinOp": self.process_binary_operation,
+            "ast.Add": self._process_add,
+            "ast.Sub": self._process_subtract,
+            "ast.Mult": self._process_multiply,
+            "ast.Pow": self._process_power,
+            "ast.Div": self._process_divide,
+            "ast.USub": self._process_unary_subtract,
+            "ast.Eq": self._process_equals_to,
+            "ast.LtE": self._process_less_than_or_equal_to,
+            "ast.Lt": self._process_less_than,
+            "ast.Gt": self._process_greater_than,
+            "ast.GtE": self._process_greater_than_or_equal_to,
+            "ast.And": self._process_and,
+            "ast.Or": self._process_or,
+            "ast.Expr": self.process_expression,
+            "ast.Compare": self.process_compare,
+            "ast.Subscript": self.process_subscript,
+            "ast.Name": self._process_name,
+            "ast.AnnAssign": self.process_annotated_assign,
+            "ast.Assign": self.process_assign,
+            "ast.Call": self.process_call,
+            "ast.Import": self.process_import,
+            "ast.alias": self._process_alias,
+            "ast.Module": self.process_module,
+            "ast.BoolOp": self.process_boolean_operation,
+            "ast.Attribute": self._process_attribute,
+            "ast.AST": self.process_ast,
+            "ast.Tuple": self.process_tuple,
+            "ast.NameConstant": self.process_name_constant,
+        }
+
+    def generate_code(self, node, state):
+        """
+            This function parses the ast node of the python file and generates
```

Comment:
See previous comment about docstring indentation - hopefully cherry-picking the commit referenced earlier will fix this file as well.